examples/README.md

```diff
@@ -164,6 +164,17 @@ cargo run --example=gauge --features=crossterm
 
 ![Gauge][gauge.gif]
 
+## Line Gauge
+
+Demonstrates the [`Line Gauge`](https://docs.rs/ratatui/latest/ratatui/widgets/struct.LineGauge.html) widget.
+Source: [line_gauge.rs](./line_gauge.rs).
+
+```shell
+cargo run --example=line_gauge --features=crossterm
+```
+
+![LineGauge][line_gauge.gif]
```

Comment:
Also needs to link the exact file in the bottom of the file.

(I don't like this Markdown feature as it makes it easy to forget… I would prefer `![LineGauge](./line_gauge.gif)` and be done with it.)